Core/src/org/sleuthkit/autopsy/datamodel/BlackboardArtifactNode.java

```diff
@@ -64,6 +67,13 @@
 public class BlackboardArtifactNode extends DisplayableItemNode {
 
     private static final Logger LOGGER = Logger.getLogger(BlackboardArtifactNode.class.getName());
+    private static final Set<String> CASE_EVENTS_OF_INTEREST = new HashSet<>(Arrays.asList(new String[]{
```

Comment:
In the next story I'm working on I'm adding a new API to case that will take these events as a Set<Event> instead of Strings so the line referenced here will become:
    private static final Set<Case.Events> CASE_EVENTS_OF_INTEREST = EnumSet.of(Case.Events.BLACKBOARD_ARTIFACT_TAG_ADDED,
        Case.Events.BLACKBOARD_ARTIFACT_TAG_DELETED,
        Case.Events.CONTENT_TAG_ADDED,
        Case.Events.CONTENT_TAG_DELETED);